### ISSUE TITLE
docker version: force-synthmix "/usr/bin/env: ‘python’: No such file or directory"

### ISSUE THREAD

**Issue by thielfab** [davidfrantz/force#169]
When I try to call `force-synthmix` I get the following error:
```
$ force-synthmix
/usr/bin/env: ‘python’: No such file or directory
```

Checking python version + path inside container:
```
$ singularity shell force_3.7.6.sif
Singularity> python
bash: python: command not found
Singularity> python3
Python 3.8.10 (default, Sep 28 2021, 16:10:42)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.executable
'/usr/bin/python3'
```

Wrong hashbang at? https://github.com/davidfrantz/force/blob/a7162b2e665fcc05d794420afa2c6fa156b1eb10/python/force-synthmix.py#L1 

- FORCE version: docker://davidfrantz/force:3.7.6
- Running on Ubuntu Server 21.10 with singularity 3.8.2 (also observed locally with docker + Ubuntu WSL2)

**Comment by davidfrantz** [comment]
yeah, seems so... I wonder when this changed... Changing the shebang should do the trick though. I will prepare something

Thanks Fabian

**Comment by davidfrantz** [comment]
Hi @thielfab,

this is now fixed with 3.7.7.

Cheers,
David
